src/test/functional/steps/court-types.ts

```diff
@@ -53,17 +50,67 @@ When('I uncheck a court type', async () => {
 });
 
 Then('a court types error message is displayed', async () => {
-  const elementExist = await I.checkElement('#courtTypesTab .govuk-error-summary');
+  const elementExist = await I.checkElement('#courtTypesContent > div.govuk-error-summary > div > ul > li');
   expect(elementExist).equal(true);
-
 });
 
 When('I check a court type which has code associated with it', async () => {
   const selector = '#court_types';
   const elementExist = await I.checkElement('#court_types');
   expect(elementExist).equal(true);
-
   await I.click(selector);
+});
+
+Then('I will make sure that one of the court type is selected', async () => {
+  const selector = '#court_types-2';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
 
+  const elementChecked = await I.isElementChecked(selector);
+  if (!elementChecked) {
+    await I.click(selector);
+  }
 });
 
+Then('I will clear the existing gbs code and enter new the one {string}', async (gbCode: string) => {
+  const selector = 'input[name=gbsCode]';
+  const elementExist = await I.checkElement('input[name=gbsCode]');
```

Comment:
input[name=gbsCode] is used more than once, consider putting it into a variable and reusing it

Reply:
fixed